numpy/random/mtrand/mtrand.pyx

```diff
@@ -3698,7 +3698,8 @@ cdef class RandomState:
         ----------
         n : int or array_like of ints
             Parameter of the distribution, >= 0. Floats are also accepted,
-            but they will be truncated to integers.
+            but they will be truncated to integers. All values must be less
+            than or equal to np.iinfo(np.long).max
```

Comment:
`np.long` is not the same as NPY_LONG. the former is simply the built-in `long`, or `int` in python 3